setup.py

```diff
@@ -33,7 +33,7 @@
     extras_require={
         'docs': ['sphinx', 'sphinx-copybutton', 'furo'],
         'lint': ['black', 'flake8', 'mypy'],
-        'test': ['pytest', 'pytest-cov'],
+        'test': ['pytest', 'pytest-cov', 'tox'],
```

Comment:
Normally tox is not needed to run the test suite but to orchestrate it. This will also make tox install itself when running the test suite.